### DISCUSSION TITLE
Dashboards with Shiny for R, having a different sidebar for each page

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#11064] [Q&A]
### Description

Can I have a different sidebar for each page when doing a dashboard with shiny in R?

**Comment by mcanouil** [comment]
Did you try setting one for each page?

**Reply by cscheid** [reply]
I think they mean "pages" as in these pages: https://quarto.org/docs/dashboards/layout.html#pages, in which case I don't think this is possible.

**Reply by mcanouil** [reply]
I would say, it should work: https://quarto.org/docs/dashboards/inputs.html#sidebars

**Reply by cscheid** [reply]
Ah, sorry - I was thinking of "sidebars" as in the website sidebars we have and configure with YAML. Thanks for clarifying.

**Comment by ManuelSpinola** [comment]
Yes, for a dashboard (and shiny) with multiple pages.  I was trying with my
dataset but I couldn't.  I will try with the link you provided.

Manuel

El lun, 14 oct 2024 a las 19:34, Mickaël Canouil ***@***.***>)
escribió:

> I would say, it should work:
> https://quarto.org/docs/dashboards/inputs.html#sidebars
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/11064#discussioncomment-10942267>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FB42N7PO44MA6KSWLR3Z3RWLTAVCNFSM6AAAAABP55O24KVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAOJUGIZDMNY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Sitio web institucional: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Sitio web personal: Sitio personal <https://mspinola-sitioweb.netlify.app>
Blog sobre Ciencia de Datos: Blog de Ciencia de Datos
<https://mspinola-ciencia-de-datos.netlify.app>
